book/references.bib

```diff
@@ -198,4 +198,15 @@ @article{Leppanen2018
 pages = {404}, 
 number = {11}, 
 volume = {8}
+}
+
+@article{Calonne202, 
```

Comment:
@mikedurand this needs to be "Calonne2020"

Reply:
argh sorry Scott! Fixed and pushed.